quickwit/quickwit-rest-client/src/models.rs

```diff
@@ -71,6 +73,20 @@ impl ApiResponse {
     }
 }
 
+/// A cousin of [`quickwit_search::SearchResponseRest`] that implements [`Deserialize`]
+///
+/// This version of the response is necessary because
+/// `serde_json_borrow::OwnedValue` is not deserializeable.
```

Comment:
ah we can ask @PSeitz to make it Deserializable.

Reply:
`OwnedValue` is a little bit special, because it needs to to take ownership of the original `String`

Reply:
couldn't it be possible to still make it deserializable by keeping a state in the deserializer, and building a (non-original) `String`

Reply:
Yes, that should be possible, but it would require to have a special deserialization logic. Currently it's just a thin wrapper